common/src/main/java/com/microsoft/identity/common/internal/logging/RequestContext.java

```diff
@@ -22,18 +22,7 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.logging;
 
-import com.google.gson.Gson;
-
-import java.util.HashMap;
-
-// TODO I'm not wedded to this name, but the concept may work for tracking correlationIds
-public class RequestContext extends HashMap<String, String> implements IRequestContext {
-
-    private static final long serialVersionUID = -1128593786133166935L;
-    private final Gson mGson = new Gson();
-
-    @Override
-    public String toJsonString() {
```

Comment:
You might still need this method to exist.

Reply:
How come?

Reply:
This one was moved to the base class right?

Reply:
That's right - the method will be accessible from the base class (and accordingly, also available on the subclass)

Reply:
I missed the extends clause.  We might want to log, though.

Reply:
e27764e2e1c9dcf20bd4d009e31c2c396159f33e